client/my-sites/site-settings/settings-performance/main.jsx

```diff
@@ -137,14 +137,7 @@ class SiteSettingsPerformance extends Component {
 				{ siteIsJetpackNonAtomic || ( siteIsAtomic && hasManagePluginsFeature ) ? (
 					<AmpJetpack />
 				) : (
-					<AmpWpcom
-						submitForm={ submitForm }
-						trackEvent={ trackEvent }
-						updateFields={ updateFields }
-						isSavingSettings={ isSavingSettings }
-						isRequestingSettings={ isRequestingSettings }
-						fields={ fields }
-					/>
+					<AmpWpcom />
```

Comment:
Should we remove this entirely?

Reply:
@danielbachhuber This is how this works at the moment:

* <AmpWpcom /> is a component that is essentially going to display the upgrade notice for installing an AMP plugin and it makes it work on all WP.com sites that are not Atomic or private / coming soon / unlaunched;
* <AmpJetpack /> is a component for all Jetpack and Atomic sites: it works a bit differently e.g. if the plugin is not installed, it has a message about installing it; if it is installed, it has a message about activating it etc. 

Both of these components are related to AMP plugin that can be installed externally and not to the AMP option that was offered as a built-in option on WP.com.

So to answer your question: we can remove it if we don't want an upgrade nudge on WP.com Simple sites. I think we should not touch the Jetpack part (we can remove the display condition for Atomic if we want to). But the question is **do we want to remove the upgrade nudge for installing the AMP plugin or not**?

I would personally leave it as is because a) it is consistent with other upgrade nudges and b) it is a notice about the AMP plugin and not the built-in AMP features on WP.com so my understanding is that it would be fine?

But happy to remove it if that's your preference.

Reply:
@katinthehatsite Yes, I think we should remove the upgrade nudge entirely. AMP is more or less dead.

Reply:
@danielbachhuber - sounds good, on it 👍

Reply:
@danielbachhuber - files removed and testing instructions updated. Let me know what you think